src/Control/HTTP.php

```diff
@@ -509,6 +510,37 @@ public static function augmentState(HTTPRequest $request, HTTPResponse $response
             $cacheControlMiddleware->addVary($configVary);
         }
 
+        // Pass cache_control to middleware
+        $configCacheControl = $config->get('cache_control');
+        if ($configCacheControl) {
+            Deprecation::notice('5.0', 'Use HTTPCacheControlMiddleware API instead');
+
+            $supportedDirectives = ['max-age', 'no-cache', 'no-store', 'must-revalidate'];
+            if ($foundUnsupported = array_diff(array_keys($configCacheControl), $supportedDirectives)) {
+                throw new \LogicException(
```

Comment:
Could you add this to the PHPDoc please?

Reply:
Done